lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -207,7 +219,42 @@ newStakePoolLayer
             & view #retirement
             & fmap (view (#epochNumber . #getApiT))
 
-    getTip = liftIO $ unsafeRunExceptT currentNodeTip
```

Comment:
Not sure what this `unsafeRunExceptT` was doing here, but nothing good could come from it for sure.